src/test/kotlin/uk/gov/justice/digital/hmpps/dpssmoketest/integration/SmokeTestIntegrationTest.kt

```diff
@@ -1,41 +1,124 @@
 package uk.gov.justice.digital.hmpps.dpssmoketest.integration
 
+import org.junit.jupiter.api.Nested
 import org.junit.jupiter.api.Test
 import org.springframework.beans.factory.annotation.Autowired
-import org.springframework.http.MediaType.APPLICATION_JSON
+import org.springframework.http.MediaType.TEXT_EVENT_STREAM
+import org.springframework.test.web.reactive.server.FluxExchangeResult
+import reactor.test.StepVerifier
 import uk.gov.justice.digital.hmpps.dpssmoketest.helper.JwtAuthHelper
+import uk.gov.justice.digital.hmpps.dpssmoketest.resource.SmokeTestResource.TestResult
+import uk.gov.justice.digital.hmpps.dpssmoketest.service.CommunityService
 
 class SmokeTestIntegrationTest : IntegrationTestBase() {
 
   @Autowired
   protected lateinit var jwtAuthHelper: JwtAuthHelper
 
-  @Test
-  fun `requires valid authentication token`() {
-    webTestClient.post()
-        .uri("/smoke-test")
-        .accept(APPLICATION_JSON)
-        .exchange()
-        .expectStatus().isUnauthorized
-  }
+  @Autowired
+  private lateinit var communityService: CommunityService
+
+  @Nested
+  inner class Authentication {
+    @Test
+    fun `requires valid authentication token`() {
+      webTestClient.post()
+          .uri("/smoke-test")
+          .accept(TEXT_EVENT_STREAM)
+          .exchange()
+          .expectStatus().isUnauthorized
+    }
+
+    @Test
+    fun `requires correct role`() {
+      webTestClient.post()
+          .uri("/smoke-test")
+          .accept(TEXT_EVENT_STREAM)
+          .headers(jwtAuthHelper.setAuthorisation("dps-smoke-test", listOf()))
+          .exchange()
+          .expectStatus().isForbidden
+    }
+
+    @Test
+    fun `succeeds with correct access`() {
+      webTestClient.post()
+          .uri("/smoke-test")
+          .accept(TEXT_EVENT_STREAM)
+          .headers(jwtAuthHelper.setAuthorisation("dps-smoke-test", listOf("ROLE_SMOKE_TEST")))
+          .exchange()
+          .expectStatus().isOk
 
-  @Test
-  fun `requires correct role`() {
-    webTestClient.post()
-        .uri("/smoke-test")
-        .accept(APPLICATION_JSON)
-        .headers(jwtAuthHelper.setAuthorisation("dps-smoke-test", listOf()))
-        .exchange()
-        .expectStatus().isForbidden
+    }
   }
 
-  @Test
-  fun `succeeds with correct access`() {
-    webTestClient.post()
-        .uri("/smoke-test")
-        .accept(APPLICATION_JSON)
-        .headers(jwtAuthHelper.setAuthorisation("dps-smoke-test", listOf("ROLE_SMOKE_TEST")))
-        .exchange()
-        .expectStatus().isOk
+  @Nested
+  inner class CircleExperiments {
```

Comment:
This emulates the 3 experiments we want to run on Circle, but in a much shorter timespan.  Gives us confidence that the Circle tests *should* behave as expected.